sdk/purview/azure-purview-workflow/tests/testcase.py

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+import functools
+import os
+from devtools_testutils import AzureTestCase, PowerShellPreparer
```

Comment:
@CordeliaGrey You shall use `AzureRecordedTestCase`. Please refer to latest test guidance:
https://github.com/Azure/azure-sdk-for-python/wiki/Dataplane-Codegen-Quick-Start-for-Test#writing-new-tests

Reply:
Fixed, thanks